File: CustomAlertNotifications/CustomAlertNotifications/Form1.vb
Public Class Form1


    Public Sub Alert(msg As String, type As frmAlert.alertTypeEnum)
        Dim f As frmAlert = New frmAlert
        f.setAlert(msg, type)
    End Sub
    Private Sub GunaAdvenceButton1_Click(sender As Object, e As EventArgs) Handles GunaAdvenceButton1.Click
        Alert("Success message", frmAlert.alertTypeEnum.Success)
    End Sub

    Private Sub GunaAdvenceButton2_Click(sender As Object, e As EventArgs) Handles GunaAdvenceButton2.Click
        Alert("Warning message", frmAlert.alertTypeEnum.Warning)
    End Sub

    Private Sub GunaAdvenceButton3_Click(sender As Object, e As EventArgs) Handles GunaAdvenceButton3.Click
        Alert("Error message", frmAlert.alertTypeEnum.Error)
    End Sub

    Private Sub GunaAdvenceButton4_Click(sender As Object, e As EventArgs) Handles GunaAdvenceButton4.Click
        Alert("Info message", frmAlert.alertTypeEnum.Info)
    End Sub
End Class

File: CustomAlertNotificationsCShrap/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;
using System.Windows.Forms;

namespace CustomAlertNotificationsCShrap
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }

        public void Alert(string msg, frmAlert.alertTypeEnum type)
        {
            frmAlert f = new frmAlert();
            f.setAlert(msg, type);
        }

        private void GunaAdvenceButton1_Click(object sender, EventArgs e)
        {
            this.Alert("Success message", frmAlert.alertTypeEnum.Success);
        }

        private void GunaAdvenceButton2_Click(object sender, EventArgs e)
        {
            this.Alert("Warning message", frmAlert.alertTypeEnum.Warning);
        }

        private void GunaAdvenceButton3_Click(object sender, EventArgs e)
        {
            this.Alert("Error message", frmAlert.alertTypeEnum.Error);
        }

        private void GunaAdvenceButton4_Click(object sender, EventArgs e)
        {
            this.Alert("Info message", frmAlert.alertTypeEnum.Info);
        }
    }
}
